dataexamples.md

```diff
@@ -114,6 +114,19 @@ Example Software: R
 Author Contact: @thoughtfulnz on Twitter
 ```
 
+```
+Data Description: Data from Global Fishing Watch
+Data URL: https://globalfishingwatch.org/datasets-and-code/vessel-identity/
+Data Licence: Creative Commons Attribution-ShareAlike 4.0 International
+Example Title: Which Countries have Large Scale Fishing Vessels?
+Example Description: Analysis and map of global fishing vessels
+Example Code URL: https://kimnewzealand.github.io/2019/06/05/vessel-analysis/
+Example Code Licence: Creative Commons Attribution-ShareAlike 4.0 International
+Example Product URL: https://www.rstudio.com/
```

Comment:
@kimnewzealand the product url is more for a product you've produced rather than pointing to the product you used to produce it. This was so if your code is used to produce something you can have separate urls pointing to the source code and what the source code produces. Can you change this so that your code url here is the product url and then provide a separate url to the open source code you produced (if you have this up somewhere).